### ISSUE TITLE
Azure mssql/mariadb as the external database (moodle)

### ISSUE THREAD

**Issue by tbertenshaw** [bitnami/charts#3708]
**Which chart**:
Moodle  moodle-8.1.5 (app version 3.9.2)

**Describe the bug**
I'm trying to use the helm chart with an existing persistant volume and an external database.

When trying to use an Azure database as the external database for the chart. I get different errors shown in the moodle container logs, depending on what version or type of database i use.

https://moodle.org/mod/forum/discuss.php?d=410491

I have tried Mariadb and mysql and mssql.  (incidentally had to stop the enforce SSL option seems the helm chart doesn't allow to specify that when connecting)

**To Reproduce**
I initially tried using mariadb(10.3), but couldn't set the innodb_large_prefix variable that was being reported as a requirement in the logs. It also mentioned something about using ANTELOPE when should be using BARRACUDA as the innodb_file_format. But this doesn't seem possible to change in Azure.

using mysql i get the following error (which seems to be a mariadb version its looking for indicating a hard requirement for mariadb with this version of moodle or helm chart?)
```
Moodle 3.9.1 (Build: 20200713) command line installation program 

 == Environment == 

 !! database mariadb (5.6.42.0) !! 

 [System] version 10.2.29 is required and you are running 5.6.42.0 -
```
**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.3.2", GitCommit:"e5077257b6ca106d1f65652b4ca994736d221ab1", GitTreeState:"clean", GoVersion:"go1.14.9"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.8", GitCommit:"9f2892aab98fe339f3bd70e3c470144299398ace", GitTreeState:"clean", BuildDate:"2020-08-13T16:12:48Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6", GitCommit:"1994a5495a40a663921c5ecfee7dd9a8c61704fa", GitTreeState:"clean", BuildDate:"2020-07-23T22:06:44Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**

**Comment by carrodher** [comment]
Moodle itself (as application, not the chart) has some requirements regarding the supported databases and specific versions, see https://docs.moodle.org/dev/Moodle_3.9_release_notes#Database_requirements

In the case of MariaDB, the minimum supported version is 10.2.29, while it seems you're using 5.6.X

**Comment by tbertenshaw** [comment]
seems someone else spotted this and resolved in #4540

**Comment by miguelaeh** [comment]
Hi @tbertenshaw ,
Thank you for sharing it in this thread!
